node_modules/oae-core/aboutcontent/aboutcontent.html

```diff
@@ -53,7 +53,7 @@ <h3 id="aboutcontent-title">${contentProfile.displayName|encodeForHTML}</h3>
             </div>
             {if contentProfile.resourceSubType === 'link'}
                 <div id="aboutcontent-link">
-                    <a href="${contentProfile.link}" title="${contentProfile.displayName|encodeForHTMLAttribute}" target="_blank">${contentProfile.link|encodeForHTML}</a>
+                    <a href="${contentProfile.link}" target="_blank">${contentProfile.link|encodeForHTML}</a>
```

Comment:
This title attribute isn't entirely redundant as it provides a readable tooltip for the link